### ISSUE TITLE
Script Extender is throwing off error messages and will not allow game to function - is Update in the works?  

### ISSUE THREAD

**Issue by DaelonDuluc** [Norbyte/bg3se#182]
Post today's update receiving attached
![SciptExtenderErrorMsg](https://github.com/Norbyte/bg3se/assets/149717338/91addd71-c26b-4a09-9f88-dbef8b017d52)

Let me know if I need to do anything.

Thanks!

**Comment by Guthixian** [comment]
Same problem here, uninstalling all my mods does nothing so the script extender is the culprit

**Comment by DaelonDuluc** [comment]
just looked at the faq's and Patch 5 is not yet supported.  That's cool.  The team will get this solved soon.  I can always depend on them.  :)

**Comment by shunshine84** [comment]
Same. They're usually pretty quick to update after a new patch so I'm just going to wait it out :D

**Comment by dekayden** [comment]
How long does this usually take to update? Day or two?

**Comment by aaronchance** [comment]
At least a week, maybe more depending on the patch size, and this one was a doozy

**Comment by Bortolosog** [comment]
I would like to know if it takes more than 1 day, if so I will have to continue without the mods :(

**Comment by Gilliestrated** [comment]
For those of us that use Vortex, how long can we expect until we can actually play with mods? thanks

**Comment by nicolasDmit** [comment]
At last time it's take 3 days, as i remember

**Comment by aaronchance** [comment]
Really? I'm pretty sure it was at least a week.

**Comment by DaelonDuluc** [comment]
maybe a day or two?  Last time it took about 24 hours for me (Patch 4).  

It's cool.  They do such a great job and do all of this for free that I'm happy to be grateful and patient.

I have other games. :)

**Comment by FallenStar08** [comment]
Before anyone jumps on me I completely understand that people have lives and have other priorities in their lives and making mods for the most part is a hobby for most. I was just wondering how close or far the devs might be to a fix? 2 hours isnt so bad, but going on 3 hours and no word is tense.

I do not want to harass the Devs or anyone directly connected to this mod, thats not my intention. A small even "where not done yet" is fine enough lol. I really didn't notice how many mods actually relied on this as a dependency until it broke in the latest update. Rolling back an update is obviously still an option and one I'm still considering but want to wait until I can get a rough timeline on this update.

I hope all the devs are doing well with the greatest health, I want to also thank them for making this community even possible so quickly, literally without there discoveries (alongside the community) the modding landscape for this game wouldve been so very dry until Larian released actual mod tools. So again with the fullest of my heart thank you Norbyte (and team if there is one) and the community uses of the mods and mod devs using the mod, thank you as well. Like me you are most likely as anxious for this update as well.

Also, is Norbyte a commercial entity?

**Comment by DaelonDuluc** [comment]
> Before anyone jumps on me I completely understand that people have lives and have other priorities in their lives and making mods for the most part is a hobby for most. I was just wondering how close or far the devs might be to a fix? 2 hours isnt so bad, but going on 3 hours and no word is tense.
> 
> I do not want to harass the Devs or anyone directly connected to this mod, thats not my intention. A small even "where not done yet" is fine enough lol. I really didn't notice how many mods actually relied on this as a dependency until it broke in the latest update. Rolling back an update is obviously still an option and one I'm still considering but want to wait until I can get a rough timeline on this update.
> 
> I hope all the devs are doing well with the greatest health, I want to also thank them for making this community even possible so quickly, literally without there discoveries (alongside the community) the modding landscape for this game wouldve been so very dry until Larian released actual mod tools. So again with the fullest of my heart thank you Norbyte (and team if there is one) and the community uses of the mods and mod devs using the mod, thank you as well. Like me you are most likely as anxious for this update as well.
> 
> Also, is Norbyte a commercial entity?

It may take a day or two so be patient.  that's what it took for Patch 5.  It's not unusual for games of this scope and with as many mods as touched by this .dll to take a little bit.

They'll get it done.  It might take a day or so, but I'm willing to be patient.  I'm just grateful they're even willing to do it!

**Comment by dekayden** [comment]
How do you roll back the update? I wouldn't my rolling back to patch 4 and playing will I wait on the Se update

**Comment by DaelonDuluc** [comment]
> In the meantime, you can try to remove the DWrite.dll file from the bin folder. I managed to load my saves that way, though naturally, there will be a few mods which won't work properly without the script extender

True... true.  Good thought though!

**Comment by DaelonDuluc** [comment]
> How do you roll back the update? I wouldn't my rolling back to patch 4 and playing will I wait on the Se update

I don't think you can.

**Comment by vanitydecay** [comment]
With how popular the game is, and all the mods that require the scrip extender, I'm sure everything will be fixed soon. Looking forward to seeing the next update!

**Comment by jeremyglebe** [comment]
> I tried to delete my mod that uses the Script Extender, but I can't log in to my last save, there's no way, right? T-T

Some mods definitely break saves. Try looking at the mod's page and see if they have any instructions for safely deleting. Many of them do. It is one downside to modding.

**Comment by vanitydecay** [comment]
> I tried to delete my mod that uses the Script Extender, but I can't log in to my last save, there's no way, right? T-T

It's definitely because of the SE! I tried logging into a save as well and it just wouldn't load, then I attempted to start a new game and it gave me the same crash as the original post. We're in the same boat!

**Comment by Macro404** [comment]
> > How do you roll back the update? I wouldn't my rolling back to patch 4 and playing will I wait on the Se update
> 
> I don't think you can.

You can, right click bg3 to open properties -> Betas -> Select patch 4 in the dropdown for Beta Participation.

**Comment by Lightboom9** [comment]
> > Before I express any concerns, I want to acknowledge that individuals lead busy lives with various priorities, and modding is often a hobby for many. My inquiry is simply aimed at understanding the proximity or distance of the developers to resolving the issue. A two-hour delay is manageable, but as we approach the three-hour mark without any updates, there's a sense of unease.
> > I want to emphasize that my intent is not to harass the developers or anyone associated with the mod directly. Even a brief acknowledgment like "we're still working on it" would suffice. It's only now apparent to me how many mods rely on this particular one as a dependency, especially considering its disruption in the latest update. While rolling back the update remains an option I'm considering, I prefer to wait until I have a rough timeline for the resolution.
> > I extend my well wishes to all the developers, hoping for their utmost health. I also want to express gratitude for swiftly creating this community. Without their contributions, alongside the collaborative efforts of the community, the modding scene for this game would have been considerably lacking until Larian released official mod tools. Therefore, from the depths of my heart, thank you to Norbyte and any associated team, as well as the users and developers contributing to the mod. Like myself, I imagine many share the anticipation for this update.
> > Thank you, and may peace and love prevail.
> 
> Made two accounts to say this, huh? There is nothing concerning about 3 hours. It is a mod. They don't have access to the update any earlier than we do. It isn't officially supported by Larian. Just chill and give people time to look at what all changed in the update and what they need to do to fix it. For all the repeated statement about how you aren't harassing them, this is still a _very_ impatient comment.

I agree. This attempt at bullying the creater of, perhaps, the most important mod for BG3 is simply disgusting.

**Comment by DaelonDuluc** [comment]
> > > How do you roll back the update? I wouldn't my rolling back to patch 4 and playing will I wait on the Se update
> > 
> > 
> > I don't think you can.
> 
> You can, right click bg3 to open properties -> Betas -> Select patch 4 in the dropdown for Beta Participation.

Cool!  I did not know that!

**Comment by Macro404** [comment]
> > > > How do you roll back the update? I wouldn't my rolling back to patch 4 and playing will I wait on the Se update
> > > 
> > > 
> > > I don't think you can.
> > 
> > 
> > You can, right click bg3 to open properties -> Betas -> Select patch 4 in the dropdown for Beta Participation.
> 
> Cool! I did not know that!

In early access they would print a guide for it every time they patched due to the updates wiping all saves.

**Comment by kweeky** [comment]
> > > How do you roll back the update? I wouldn't my rolling back to patch 4 and playing will I wait on the Se update
> > 
> > 
> > I don't think you can.
> 
> You can, right click bg3 to open properties -> Betas -> Select patch 4 in the dropdown for Beta Participation.

This is excellent, good on Larian for providing this. It means people who desperately want to carry on playing with bg3se right now can until it's updated!

**Comment by Bortolosog** [comment]
> > > > > How do you roll back the update? I wouldn't my rolling back to patch 4 and playing will I wait on the Se update
> > > > 
> > > > 
> > > > I don't think you can.
> > > 
> > > 
> > > You can, right click bg3 to open properties -> Betas -> Select patch 4 in the dropdown for Beta Participation.
> > 
> > 
> > Cool! I did not know that!
> 
> In early access they would print a guide for it every time they patched due to the updates wiping all saves.


When the extender script updates, just download it? It won't interfere with game progression, will I keep it in patch 4?

**Comment by Bortolosog** [comment]
> > > How do you roll back the update? I wouldn't my rolling back to patch 4 and playing will I wait on the Se update
> > 
> > 
> > I don't think you can.
> 
> You can, right click bg3 to open properties -> Betas -> Select patch 4 in the dropdown for Beta Participation.

THANK YOUUUUUUUUUUUUUUUUU, <3

**Comment by DaelonDuluc** [comment]
> > > > How do you roll back the update? I wouldn't my rolling back to patch 4 and playing will I wait on the Se update
> > > 
> > > 
> > > I don't think you can.
> > 
> > 
> > You can, right click bg3 to open properties -> Betas -> Select patch 4 in the dropdown for Beta Participation.
> 
> Cool! I did not know that!

Where do I right-click? In Steam? (don't see the option).  In Larian or in the actual files on my PC?  Sorry - not very good at this whole PC thing yet.

**Comment by FallenStar08** [comment]
The creator could have updated the SE before patch5 dropped, there are no excuse to have to wait THAT long for our mods we paid for to work.

**Comment by Macro404** [comment]
> > > > > How do you roll back the update? I wouldn't my rolling back to patch 4 and playing will I wait on the Se update
> > > > 
> > > > 
> > > > I don't think you can.
> > > 
> > > 
> > > You can, right click bg3 to open properties -> Betas -> Select patch 4 in the dropdown for Beta Participation.
> > 
> > 
> > Cool! I did not know that!
> 
> Where do I right-click? In Steam? (don't see the option). In Larian or in the actual files on my PC? Sorry - not very good at this whole PC thing yet.

Right click bg3 where your games are listed in your steam library.

**Comment by DaelonDuluc** [comment]
> > > > > > How do you roll back the update? I wouldn't my rolling back to patch 4 and playing will I wait on the Se update
> > > > > 
> > > > > 
> > > > > I don't think you can.
> > > > 
> > > > 
> > > > You can, right click bg3 to open properties -> Betas -> Select patch 4 in the dropdown for Beta Participation.
> > > 
> > > 
> > > Cool! I did not know that!
> > 
> > 
> > Where do I right-click? In Steam? (don't see the option). In Larian or in the actual files on my PC? Sorry - not very good at this whole PC thing yet.
> 
> Right click bg3 where your games are listed in your steam library.

Thank you!

**Comment by callistowrites** [comment]
> The creator could have updated the SE before patch5 dropped, there are no excuses to have to wait THAT long for our mods we paid for to work.

I have not paid for a single mod. And as another said, since the SE is not created by Larian, we do have to wait. SE's creators don't get the patch information any earlier than we do.

**Comment by decamhian** [comment]
Is Norbyte a commercial entity?

**Comment by Gilliestrated** [comment]
> > > > > > How do you roll back the update? I wouldn't my rolling back to patch 4 and playing will I wait on the Se update
> > > > > 
> > > > > 
> > > > > I don't think you can.
> > > > 
> > > > 
> > > > You can, right click bg3 to open properties -> Betas -> Select patch 4 in the dropdown for Beta Participation.
> > > 
> > > 
> > > Cool! I did not know that!
> > 
> > 
> > In early access they would print a guide for it every time they patched due to the updates wiping all saves.
> 
> When the extender script updates, just download it? It won't interfere with game progression, will I keep it in patch 4?

I would like to know this as well ^^^^^

**Comment by Lightboom9** [comment]
> > > Before I express any concerns, I want to acknowledge that individuals lead busy lives with various priorities, and modding is often a hobby for many. My inquiry is simply aimed at understanding the proximity or distance of the developers to resolving the issue. A two-hour delay is manageable, but as we approach the three-hour mark without any updates, there's a sense of unease.
> > > I want to emphasize that my intent is not to harass the developers or anyone associated with the mod directly. Even a brief acknowledgment like "we're still working on it" would suffice. It's only now apparent to me how many mods rely on this particular one as a dependency, especially considering its disruption in the latest update. While rolling back the update remains an option I'm considering, I prefer to wait until I have a rough timeline for the resolution.
> > > I extend my well wishes to all the developers, hoping for their utmost health. I also want to express gratitude for swiftly creating this community. Without their contributions, alongside the collaborative efforts of the community, the modding scene for this game would have been considerably lacking until Larian released official mod tools. Therefore, from the depths of my heart, thank you to Norbyte and any associated team, as well as the users and developers contributing to the mod. Like myself, I imagine many share the anticipation for this update.
> > > Thank you, and may peace and love prevail.
> > 
> > 
> > Made two accounts to say this, huh? There is nothing concerning about 3 hours. It is a mod. They don't have access to the update any earlier than we do. It isn't officially supported by Larian. Just chill and give people time to look at what all changed in the update and what they need to do to fix it. For all the repeated statement about how you aren't harassing them, this is still a _very_ impatient comment.
> 
> I love how they try to make it seem like they aren't rushing anything and they get that this isn't a main job for modders, but then proceeds to complain about a couple hours of waiting. Maybe they are at work & didn't even see that the patch updated. I personally didn't even see the patch updated until an hour ago. 3 hours is apparently unacceptable lol The entitlement is weird.

I kind of agree. However, it's now have been 4 hours. It's becoming unacceptable.

**Comment by dekayden** [comment]
> The creator could have updated the SE before patch5 dropped, there are no excuses to have to wait THAT long for our mods we paid for to work.

Please tell me you are joking? Hahaha you can't be serious.

**Comment by Bortolosog** [comment]
When the extender script updates, just download it? It won't interfere with game progression, will I keep it in patch 4??

**Comment by Macro404** [comment]
> > > > > > > How do you roll back the update? I wouldn't my rolling back to patch 4 and playing will I wait on the Se update
> > > > > > 
> > > > > > 
> > > > > > I don't think you can.
> > > > > 
> > > > > 
> > > > > You can, right click bg3 to open properties -> Betas -> Select patch 4 in the dropdown for Beta Participation.
> > > > 
> > > > 
> > > > Cool! I did not know that!
> > > 
> > > 
> > > In early access they would print a guide for it every time they patched due to the updates wiping all saves.
> > 
> > 
> > When the extender script updates, just download it? It won't interfere with game progression, will I keep it in patch 4?
> 
> I would like to know this as well ^^^^^

That comment about wiping saves was due to Larian changing the story during early access, it should not be a factor anymore.

**Comment by shunshine84** [comment]
> I kind of agree. However, it's now have been 4 hours. It's becoming unacceptable.

I'm convinced you're trolling at this point.

**Comment by dekayden** [comment]
> When the extender script updates, just download it? It won't interfere with game progression, will I keep it in patch 4??

So if I understand correctly the SE has an auto update features built in. However I would redownload it to be safe. I will be redownloading it as soon as they give us the all clear.

**Comment by FallenStar08** [comment]
The thing I don't understand is my SE auto updates when I launch the game, so why can't the creator launch the game to auto update his SE????

**Comment by Macro404** [comment]
> The thing I don't understand is my SE auto updates when I launch the game, so why can't the creator launch the game to auto update his SE????

Because the latest update isn't compatible with patch 5. Once it is compatible they will release an update and your SE will automatically fetch that update.

**Comment by shunshine84** [comment]
> > When the extender script updates, just download it? It won't interfere with game progression, will I keep it in patch 4??
> 
> So if I understand correctly the SE has an auto update features built in. However I would redownload it to be safe. I will be redownloading it as soon as they give us the all clear.

Correct. You only have to redownload the script extender when they have a new download version of it on their page. The last one was upgraded on 10/28. I did not have to redownload with the last patch.

**Comment by dekayden** [comment]
> > > When the extender script updates, just download it? It won't interfere with game progression, will I keep it in patch 4??
> > 
> > 
> > So if I understand correctly the SE has an auto update features built in. However I would redownload it to be safe. I will be redownloading it as soon as they give us the all clear.
> 
> Correct. You only have to redownload the script extender when they have a new download version of it on their page. The last one was upgraded on 10/28. I did not have to redownload with the last patch.

That's what I figured. I just wanted to be safe than sorry haha

**Comment by Bortolosog** [comment]
Guys, why when I click to go back to patch 4, a big update starts? (1 hour) wasn't I playing this version until yesterday? sorry for the stupid question... 
![image](https://github.com/Norbyte/bg3se/assets/152534602/7d6a25b2-63c1-45af-9311-d4a42d70a0eb)

**Comment by dekayden** [comment]
Also I love how some of these people are crying at how long it is taking the Mod team to work on correcting this.

**Comment by dekayden** [comment]
> Guys, why when I click to go back to patch 4, a big update starts? (1 hour) wasn't I playing this version until yesterday? sorry for the stupid question... ![image](https://private-user-images.githubusercontent.com/152534602/287058083-7d6a25b2-63c1-45af-9311-d4a42d70a0eb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDEzNzU2NDMsIm5iZiI6MTcwMTM3NTM0MywicGF0aCI6Ii8xNTI1MzQ2MDIvMjg3MDU4MDgzLTdkNmEyNWIyLTYzYzEtNDVhZi05MzExLWQ0YTQyZDcwYTBlYi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMTMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTEzMFQyMDE1NDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yZTU4MDkyYzJkOGUzMmIyODMzYWVkZTE2NDNhYzVjMzBjODZiNjcwOGIwZDU5OGM1YmNmOWE1NmM4MTc4NWIxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.AlkRhsqqyEK91s1eQ-PMmYncnZ-oGz5VlAgEnWcvlHo)

That is your game rolling back via disk speed. I too am going through a validation process to roll back to Patch 4

**Comment by Gilliestrated** [comment]
> > > > > > > > How do you roll back the update? I wouldn't my rolling back to patch 4 and playing will I wait on the Se update
> > > > > > > 
> > > > > > > 
> > > > > > > I don't think you can.
> > > > > > 
> > > > > > 
> > > > > > You can, right click bg3 to open properties -> Betas -> Select patch 4 in the dropdown for Beta Participation.
> > > > > 
> > > > > 
> > > > > Cool! I did not know that!
> > > > 
> > > > 
> > > > In early access they would print a guide for it every time they patched due to the updates wiping all saves.
> > > 
> > > 
> > > When the extender script updates, just download it? It won't interfere with game progression, will I keep it in patch 4?
> > 
> > 
> > I would like to know this as well ^^^^^
> 
> That comment about wiping saves was due to Larian changing the story during early access, it should not be a factor anymore.

I updated the game to beta patch 4 and it will not let me load any of my previous saves. says all mods are out of date and wont be loaded.

**Comment by dekayden** [comment]
I think in the future I will turn off Auto Update for this game seeing how mostly all my saves have Mods attached to it. This way I can continue playing the game with out having to roll back till an SE patch.

**Comment by Zwiecho** [comment]
> > > > > > > > > > How do you roll back the update? I wouldn't my rolling back to patch 4 and playing will I wait on the Se update
> > > > > > > > > 
> > > > > > > > > 
> > > > > > > > > I don't think you can.
> > > > > > > > 
> > > > > > > > 
> > > > > > > > You can, right click bg3 to open properties -> Betas -> Select patch 4 in the dropdown for Beta Participation.
> > > > > > > 
> > > > > > > 
> > > > > > > Cool! I did not know that!
> > > > > > 
> > > > > > 
> > > > > > In early access they would print a guide for it every time they patched due to the updates wiping all saves.
> > > > > 
> > > > > 
> > > > > When the extender script updates, just download it? It won't interfere with game progression, will I keep it in patch 4?
> > > > 
> > > > 
> > > > I would like to know this as well ^^^^^
> > > 
> > > 
> > > That comment about wiping saves was due to Larian changing the story during early access, it should not be a factor anymore.
> > 
> > 
> > I updated the game to beta patch 4 and it will not let me load any of my previous saves. says all mods are out of date and wont be loaded.
> 
> This is concerning. I haven't made it that far but as soon as I can attempt to load up the patch 4 game I will get back to this and let you know what I find.

I believe that when you look at the version you are rolling back to, you can read "...patch_hf1", so it might roll you back to hotfix 1 of patch 4, so a much earlier version than the one you had just before Patch 5 rolled out, which may cause the inability to load recent saves...

**Comment by Es-we3** [comment]
> > > > > > > > > > > How do you roll back the update? I wouldn't my rolling back to patch 4 and playing will I wait on the Se update
> > > > > > > > > > 
> > > > > > > > > > 
> > > > > > > > > > I don't think you can.
> > > > > > > > > 
> > > > > > > > > 
> > > > > > > > > You can, right click bg3 to open properties -> Betas -> Select patch 4 in the dropdown for Beta Participation.
> > > > > > > > 
> > > > > > > > 
> > > > > > > > Cool! I did not know that!
> > > > > > > 
> > > > > > > 
> > > > > > > In early access they would print a guide for it every time they patched due to the updates wiping all saves.
> > > > > > 
> > > > > > 
> > > > > > When the extender script updates, just download it? It won't interfere with game progression, will I keep it in patch 4?
> > > > > 
> > > > > 
> > > > > I would like to know this as well ^^^^^
> > > > 
> > > > 
> > > > That comment about wiping saves was due to Larian changing the story during early access, it should not be a factor anymore.
> > > 
> > > 
> > > I updated the game to beta patch 4 and it will not let me load any of my previous saves. says all mods are out of date and wont be loaded.
> > 
> > 
> > This is concerning. I haven't made it that far but as soon as I can attempt to load up the patch 4 game I will get back to this and let you know what I find.
> 
> I believe that when you look at the version you are rolling back to, you can read "...patch4_hf1", so it might roll you back to hotfix 1 of patch 4, so a much earlier version than the one you had just before Patch 5 rolled out, which may cause the inability to load recent saves...

I'm downloading the Patch 4, I hope it's not as you say, but if it is, when I go back to patch 5, the saves will be back?

**Comment by DaelonDuluc** [comment]
> > > > > > > > > > > > How do you roll back the update? I wouldn't my rolling back to patch 4 and playing will I wait on the Se update
> > > > > > > > > > > 
> > > > > > > > > > > 
> > > > > > > > > > > I don't think you can.
> > > > > > > > > > 
> > > > > > > > > > 
> > > > > > > > > > You can, right click bg3 to open properties -> Betas -> Select patch 4 in the dropdown for Beta Participation.
> > > > > > > > > 
> > > > > > > > > 
> > > > > > > > > Cool! I did not know that!
> > > > > > > > 
> > > > > > > > 
> > > > > > > > In early access they would print a guide for it every time they patched due to the updates wiping all saves.
> > > > > > > 
> > > > > > > 
> > > > > > > When the extender script updates, just download it? It won't interfere with game progression, will I keep it in patch 4?
> > > > > > 
> > > > > > 
> > > > > > I would like to know this as well ^^^^^
> > > > > 
> > > > > 
> > > > > That comment about wiping saves was due to Larian changing the story during early access, it should not be a factor anymore.
> > > > 
> > > > 
> > > > I updated the game to beta patch 4 and it will not let me load any of my previous saves. says all mods are out of date and wont be loaded.
> > > 
> > > 
> > > This is concerning. I haven't made it that far but as soon as I can attempt to load up the patch 4 game I will get back to this and let you know what I find.
> > 
> > 
> > I believe that when you look at the version you are rolling back to, you can read "...patch4_hf1", so it might roll you back to hotfix 1 of patch 4, so a much earlier version than the one you had just before Patch 5 rolled out, which may cause the inability to load recent saves...
> 
> I'm downloading the Patch 4, I hope it's not as you say, but if it is, when I go back to patch 5, the saves will be back?

My P4 rollback seems to be good so far, but I'm sure we'll need to update to 5 once the patch is confirmed.

**Comment by Clobahon** [comment]
> > > > > > > > > > > > > How do you roll back the update? I wouldn't my rolling back to patch 4 and playing will I wait on the Se update
> > > > > > > > > > > > 
> > > > > > > > > > > > 
> > > > > > > > > > > > I don't think you can.
> > > > > > > > > > > 
> > > > > > > > > > > 
> > > > > > > > > > > You can, right click bg3 to open properties -> Betas -> Select patch 4 in the dropdown for Beta Participation.
> > > > > > > > > > 
> > > > > > > > > > 
> > > > > > > > > > Cool! I did not know that!
> > > > > > > > > 
> > > > > > > > > 
> > > > > > > > > In early access they would print a guide for it every time they patched due to the updates wiping all saves.
> > > > > > > > 
> > > > > > > > 
> > > > > > > > When the extender script updates, just download it? It won't interfere with game progression, will I keep it in patch 4?
> > > > > > > 
> > > > > > > 
> > > > > > > I would like to know this as well ^^^^^
> > > > > > 
> > > > > > 
> > > > > > That comment about wiping saves was due to Larian changing the story during early access, it should not be a factor anymore.
> > > > > 
> > > > > 
> > > > > I updated the game to beta patch 4 and it will not let me load any of my previous saves. says all mods are out of date and wont be loaded.
> > > > 
> > > > 
> > > > This is concerning. I haven't made it that far but as soon as I can attempt to load up the patch 4 game I will get back to this and let you know what I find.
> > > 
> > > 
> > > I believe that when you look at the version you are rolling back to, you can read "...patch4_hf1", so it might roll you back to hotfix 1 of patch 4, so a much earlier version than the one you had just before Patch 5 rolled out, which may cause the inability to load recent saves...
> > 
> > 
> > I'm downloading the Patch 4, I hope it's not as you say, but if it is, when I go back to patch 5, the saves will be back?
> 
> My P4 rollback seems to be good so far, but I'm sure we'll need to update to 5 once the patch is confirmed.

Yeah so I rolled back and I'm able to luanch the game now but I can't load any recent sves due to them being an incompatible version. So I've waited on 2 installs for nothing :(

**Comment by Sargonnas77** [comment]
> My Dearest Norbyte,
> 
> I pen this humble correspondence with a heavy heart, hoping against hope that these words find thee in good health amidst the trials of these tumultuous times. It is with a spirit laden with gratitude that I extend my sincerest thanks for thy creation, the Script Extender mod.
> 
> In the throes of battle within the digital realm, thy ingenious contrivance hath served as a beacon of solace and diversion, bringing a measure of respite to these wearied virtual campaigns. Alas, the rigors of life, akin to the trials faced on the battlefield, are understood, and I pray that this epistle reaches thee in a moment of respite from such burdens.
> 
> With a sense of duty akin to that which compels me on the battlefield, I beseech thee, dear Norbyte, to consider, when time and circumstance permit, a renewal of thy efforts in updating the Script Extender mod. It is a humble entreaty, recognizing the myriad tribulations that besiege us in these trying days.
> 
> Thy labor, much like the toil of a soldier on the front lines, hath not gone unnoticed. The Script Extender stands as a testament to thy commitment and prowess in the realms of code and creation. The virtual brethren, united by the common cause of immersive gaming, hold thee in high regard.
> 
> If there be any manner in which the community may lend a hand or offer succor during these digital skirmishes, let it be known. In earnest expectation and with an ardent desire for thy renewed creative endeavors, I remain, as ever,
> 
> Yours faithfully

Like, yeah, big thank you for your work...just like last patch when way too many people freaked out about not be able to play their heavily modded fantasies.  Tomorrow, next week, whenever... seriously appreciate your effort.

**Comment by Bortolosog** [comment]
I managed to enter patch 4, with my save, is this possible?
I'm scared of moving forward and when going to 5 it won't work.


![image](https://github.com/Norbyte/bg3se/assets/152534602/27bef624-4dbb-4f4a-8198-78b30fc250c5)

**Comment by BlocTheWorker** [comment]
> I managed to enter patch 4, with my save, is this possible? I'm scared of moving forward and when going to 5 it won't work.
> 
> ![image](https://private-user-images.githubusercontent.com/152534602/287080036-27bef624-4dbb-4f4a-8198-78b30fc250c5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDEzODE2OTgsIm5iZiI6MTcwMTM4MTM5OCwicGF0aCI6Ii8xNTI1MzQ2MDIvMjg3MDgwMDM2LTI3YmVmNjI0LTRkYmItNGY0YS04MTk4LTc4YjMwZmMyNTBjNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMTMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTEzMFQyMTU2MzhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lOGZlNmUzNDdmNTNkZmQ4ZDBiZGM2YmZiNmVjOTkzYTM4ZGY1ZDBiMTk5MTk3NWE2ZWE0MWRjMjkxZTg0N2M0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.4IKV8QWWFYmbOxSBjsmWxQst459zc84h_6Iqn8YgxcM)

Patch 5 is the version 4.1.1.4061076 already. You are playing with Patch 5. See Steam for Patches and Version Numbers
![image](https://github.com/Norbyte/bg3se/assets/92448509/3557ea7b-9f27-44a2-bab0-9f347eaec98b)
---
Also, latest Script Extender version seems to be working on me - seems like Norbyte did already fix the issue. I cannot speak about others though, I have only a single mod installed on my machine

**Comment by sonofthunder77** [comment]
Hey Ash,

This is Franco, commented on your insta, twitch, and ur lives. Just wanted
to tag you on this thread from github to show you where to get real time
updates on script extender on the off chance you were not already in the
know. Looking forward to seeing you stream Patch 5!

Thanks,

FRANCO/soviet.lightning

**Comment by aeschiel** [comment]
omg, that was fast. everything seems to be working fine on my end now. thanks so much for the swift fix! <3

**Comment by LeonMorninstar** [comment]
Thank you Norbyte, you're a god

**Comment by Es-we3** [comment]
Thank you Norobyte, never seen someone work as hard nor fast, it works awesome!

**Comment by iiiisy** [comment]
I can't load my save after removing some mods (which before I could remove and open the save normally). Can any kind soul try to tell me what the problem might be?

**Comment by dajolieasya** [comment]
Dear Norbyte and team, thank you so much for your work, jeez that was fast, honour mode here I come whee

**Comment by GawDamnit** [comment]
Thanks Norbyte, you rock.

**Comment by Gilliestrated** [comment]
my Addons still dont work. do i need to re download all apps or wait for a day?

**Comment by Gilliestrated** [comment]
> my Addons still dont work. do i need to re download all apps or wait for a day?

Mods***

**Comment by Regcod** [comment]
Hi,

with only se installed I still have these errors:

******************************************************************************
*                                                                            *
*                     BG3 Script Extender Debug Console                      *
*                                                                            *
******************************************************************************

BG3Ext v9 built on Nov  5 2023 00:50:24
Game version v4.40.61.76 OK
ScriptExtender::Initialize: Starting
No match found for mapping 'EoCServer2/EoCClient2' [CRITICAL]
No match found for mapping 'ecl::GameStateMachine::Update' [CRITICAL]
No match found for mapping 'esv::GameStateThreaded::GameStateWorker::DoWork' [CRITICAL]
No match found for mapping 'esv::SavegameManager::WarnIfModded'
No match found for mapping 'esv::SavegamePostVisit_DisableStoryMerge'
No match found for mapping 'esv::StatsSystem::ThrowDamageEvent'
No match found for mapping 'RPGStats::Load'
No match found for mapping 'KillLauncher'
ScriptExtender::Initialize: Could not load libraries; skipping scripting extension initialization.
Library startup took 1111 ms
Fossilize ERROR: Failed to parse ApplicationInfoFilter, letting recording go through.
Fossilize INFO: Overriding serialization path: "G:\SteamLibrary\steamapps\shadercache\1086940\fozpipelinesv6\steamappru.
ecl::ScriptExtender::OnGameStateChanged(): Loaded module
bg3se::ExtensionStateBase::LoadConfigs(): Mod manager not available
bg3se::ExtensionStateBase::LoadConfigs(): Mod manager not available
No match found for mapping 'ls::ModuleSettings::IsModded'
LibraryManager::PostStartupFindLibraries() took 143 ms
STARTUP ERROR: A severe error has occurred during Script Extender initialization. Extension features will be unavailabl.


any clue?

ty

**Comment by Regcod** [comment]
Sorry, my mistake, forgot to change version inside updater preference... ty

**Comment by JMScottOH** [comment]
> I can't load my save after removing some mods (which before I could remove and open the save normally). Can any kind soul try to tell me what the problem might be?

I'm having the same issue post-Patch 5 and SE 11. Is this intended? It's making any game-change flexibility...well, not flexible.

**Comment by Norbyte** [comment]
I'm closing old issues related to old game versions, since those were largely fixed by later game patches and SE updates and are no longer relevant/reproducible.
